cbmc/proofs/IotMqtt_UnsubscribeAsync/README.md

```diff
@@ -0,0 +1,113 @@
+This is a memory safety proof for IotMqtt_UnsubscribeAsync.
+
+This proof attains 91% code coverage.  The following comments explain
+why the uncovered lines of code are unreachable code.
+
+Some functions contain unreachable blocks of code:
+
+* libraries/standard/common/include/iot_linear_containers.h
+  IotListDouble_FindFirstMatch
+
+  * Always called with a nonnull match predicate
+
+* libraries/standard/mqtt/src/iot_mqtt_api.c IotMqtt_OperationType
+
+  * Always called with UNSUBSCRIBE operation type
+
+* libraries/standard/mqtt/src/iot_mqtt_api.c _subscriptionCommon
```

Comment:
The markup changes to italics on the last word of this line and continues that way thereafter. I think it's because of the "_". Can you please figure out how to escape that?

Reply:
Fixed in both README.md files.